tests/integration_tests/performance/test_block_ab.py

```diff
@@ -240,3 +247,104 @@ def test_block_vhost_user_performance(
             **vm.dimensions,
         }
     )
+
+
+def _check_block_size(ssh_connection, dev_path, size):
+    """
+    Checks the size of the block device.
+    """
+    _, stdout, stderr = ssh_connection.run("blockdev --getsize64 {}".format(dev_path))
+    assert stderr == ""
+    assert stdout.strip() == str(size)
+
+
+@pytest.mark.nonci
+@pytest.mark.parametrize("vcpu_count", [1, 2], ids=["1vcpu", "2vcpu"])
+def test_vhost_user_block(microvm_factory, guest_kernel, rootfs, vcpu_count, metrics):
```

Comment:
is this a good place for this test? does it do any A/B testing or does it only emit metrics for our tracking?

Reply:
+1, if we don't A/B-test this, and only want metrics, this should be in the PR CI (so the metrics get emitted by our nightly runs). Whether we want to A/B-test I suppose depends on how much influence firecracker has on the metrics. If they are mostly backend metrics, then we probably shouldnt

Reply:
Moved the PR to draft to avoid the confusion. I am still looking for the right place. There was some code here from the last PR so this code is here no other reason.
For the A/B question, I don't think we need to have A/B, we just need to submit metrics to CW.